<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JsonToolWidget</class>
 <widget class="QWidget" name="JsonToolWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JsonToolWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QTextEdit" name="teJsonText"/>
     </item>
     <item>
      <widget class="QLabel" name="labelTips">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pbCheck">
         <property name="text">
          <string>校验</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbFormat">
         <property name="text">
          <string>格式化</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbCompact">
         <property name="text">
          <string>压缩</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
